<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CPresentationToolbar</class>
 <widget class="QWidget" name="CPresentationToolbar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>193</width>
    <height>51</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CPresentationToolbar</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <property name="spacing">
    <number>2</number>
   </property>
   <property name="leftMargin">
    <number>2</number>
   </property>
   <property name="topMargin">
    <number>2</number>
   </property>
   <property name="rightMargin">
    <number>2</number>
   </property>
   <property name="bottomMargin">
    <number>2</number>
   </property>
   <item>
    <widget class="QWidget" name="m_holder" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>2</number>
      </property>
      <property name="leftMargin">
       <number>2</number>
      </property>
      <property name="topMargin">
       <number>2</number>
      </property>
      <property name="rightMargin">
       <number>2</number>
      </property>
      <property name="bottomMargin">
       <number>2</number>
      </property>
      <item>
       <widget class="QToolButton" name="m_btnPrev">
        <property name="toolTip">
         <string>Previous Page</string>
        </property>
        <property name="text">
         <string>Prev</string>
        </property>
        <property name="icon">
         <iconset resource="../../res/mac/res.qrc">
          <normaloff>:/res/icons/presentation-prev-page.png</normaloff>:/res/icons/presentation-prev-page.png</iconset>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonTextUnderIcon</enum>
        </property>
        <property name="autoRaise">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="m_btnNext">
        <property name="toolTip">
         <string>Next Page</string>
        </property>
        <property name="text">
         <string>Next</string>
        </property>
        <property name="icon">
         <iconset resource="../../res/mac/res.qrc">
          <normaloff>:/res/icons/presentation-next-page.png</normaloff>:/res/icons/presentation-next-page.png</iconset>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonTextUnderIcon</enum>
        </property>
        <property name="autoRaise">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="m_btnClear">
        <property name="toolTip">
         <string>Clear All Notes</string>
        </property>
        <property name="text">
         <string>Clear</string>
        </property>
        <property name="icon">
         <iconset resource="../../res/mac/res.qrc">
          <normaloff>:/res/icons/presentation-clear.png</normaloff>:/res/icons/presentation-clear.png</iconset>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonTextUnderIcon</enum>
        </property>
        <property name="autoRaise">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>18</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QToolButton" name="m_btnExit">
        <property name="toolTip">
         <string>Quit Presentation</string>
        </property>
        <property name="text">
         <string>Esc</string>
        </property>
        <property name="icon">
         <iconset resource="../../res/mac/res.qrc">
          <normaloff>:/res/icons/presentation-exit.png</normaloff>:/res/icons/presentation-exit.png</iconset>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonTextUnderIcon</enum>
        </property>
        <property name="autoRaise">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../../res/mac/res.qrc"/>
  <include location="../../res/mac/res.qrc"/>
 </resources>
 <connections/>
</ui>
